Dockerfile

```
@@ -0,0 +1,80 @@
+ENV PATH="/opt/pulumi:${POETRY_HOME}/bin:/home/${USERNAME}/.local/bin:/home/${USERNAME}/.pulumi/bin:${PATH}"
+ARG POETRY_VERSION=1.8.4
+RUN bash -o pipefail -c 'curl --fail --silent --show-error --location \
+        --retry 5 --retry-delay 5 --retry-all-errors \
+        https://install.python-poetry.org \
+        | python - --version "${POETRY_VERSION}"'
+ENV POETRY_VIRTUALENVS_CREATE=false
+ENV POETRY_HTTP_TIMEOUT=60
+
+COPY --chown=${USERNAME}:${GID} pyproject.toml poetry.lock /workspace/
+
+RUN --mount=type=cache,target=/root/.cache/pip \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)